quipucords/api/messages.py

```diff
@@ -16,6 +16,8 @@
 FC_REQUIRED_ATTRIBUTE = 'Required. May not be null or empty.'
 FC_SOURCE_NAME_NOT_STR = 'Source name must be type string.'
 FC_MUST_BE_ONE_OF = 'Must be one of the following: %s'
+FC_MISSING_REPORT_VERSION = 'Missing report_version.  '\
+    'Future releases will not tolerate missing or invalid report_version.'
```

Comment:
possibly change the second sentence to `Future releases will not tolerate a missing or invalid report_version.`

Reply:
Fixed